lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Binary.php

```diff
@@ -21,9 +21,13 @@
 
 use Doctrine\Common\Annotations\Annotation;
 
+@trigger_error('The @Binary annotation is deprecated, use @Field(type="binary") instead.', E_USER_DEPRECATED);
```

Comment:
what if they are still on symfony 2.3 for example, but update phpcr-odm?

and i don't know if anybody is using phpcr-odm outside of symfony - but if we have an easy answer for them i am happy to find out :-)